content/posts/newsletter/2019-02-28.md

```diff
@@ -0,0 +1,95 @@
+---
+title: The XMPP Newsletter, 28 February 2019
+save_as: 2019/02/newsletter-28-february/index.html
+url: 2019/02/newsletter-28-february/
+date: 2019-02-28 00:00
+author: jcbrand
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter.
+
+If you have an article, tutorial or blog post you'd like us to include in the
```

Comment:
Can we, should we be a bit more "aggressive" like "publicise your XMPP-based product or services, share the news, and spread the awareness!"

Reply:
I'm not a fan of excessive use of the exclamation mark in order to show enthusiasm.

Also, people submit articles for inclusion, but we don't promise to actually include anything, since we have editorial control.